common-libs/common/src/_baseDocumentLoader/index.ts

```diff
@@ -77,18 +86,7 @@ export const baseDocumentLoader = async (url: string) => {
       return cachedDocuments.get(url)
     }
 
-    let res: Response
-
-    try {
-      res = await fetch(url, {
-        method: 'GET',
-        headers: {
-          Accept: 'application/ld+json, application/json',
-        },
-      })
-    } catch (error) {
-      throw new Error(`URL could not be dereferenced, an error occurred. ${error}`)
-    }
+    const res = await fetchSchema(url)
```

Comment:
Maybe `fetchDocument`? Or at least `fetchContext`. I don't think "schema" is the correct term here.